app/__init__.py

```diff
@@ -38,5 +44,49 @@ def new_token_route():
                 500,
             )
 
+    def Captcha_detection(image):
+
+    # write to disk
+    with open('temppiratecaptcha.png','wb') as file:
```

Comment:
Be cautious when using hardcoded file names like 'temppiratecaptcha.png.' If the application is handling multiple requests concurrently, consider using unique filenames to avoid conflicts

Reply:
This will be removed completely